index.bs

```diff
@@ -401,6 +401,25 @@ Issue(139): Clarify scoping/uniqueness of request IDs.
 Authentication {#authentication}
 ================================
 
+Each supported authentication method is implemeted via authentication messages
+specific to that method.  Authentication method is explicitly specified by
+the message itself.  Authentication status message is common for all authentication
+methods.  Any new authentication method added must define new authentication messages.
+Default authentication method is a challenge-response authentication with
+auth-request-hkdf-scrypt-psk and auth-response-hkdf-scrypt-psk-result.
+
+Prior to authentication, agents exchange auth-capabilities messages specifying
+pre-shared key ease of input for the user and supported PSK input methods.
+The agent with the lowest PSK ease of input presents PSK to the user when the agent
+either sends or receives an authentication request.  In case both agents have the same
+PSK ease of input value, the receiver presents PSK to the user.  The same pre-shared key
+is used by both agents to issue an authentication request.
+
+PSK ease of input is an integer in [0, 100] range, where 0 means it is not possible for
```

Comment:
"in the range [0, 100]"

And perhaps be explicit that about it being inclusive.